libraries/botframework-schema/src/index.ts

```diff
@@ -1753,6 +1753,7 @@ export enum ActivityTypes {
   EndOfConversation = 'endOfConversation',
   Event = 'event',
   Invoke = 'invoke',
+  InvokeResponse = 'invokeResponse',
```

Comment:
I did not see an `ActivityTypesEx` enum in the JS SDK so I just added this to the existing `ActivityTypes` enum.

Reply:
That's OK, ActivityTypesEx was created when we used to create the Activity and other classes from swagger, we don't do that anymore.